src/frontend/src/flows/displaySingleDeviceWarning.ts

```diff
@@ -0,0 +1,70 @@
+import { html, render } from "lit-html";
+import { warningIcon } from "../components/icons";
+
+const pageContent = () => html`
+  <style>
+    #warningContainer {
+      min-height: 15rem;
+    }
+    .warningIcon {
+      align-self: center;
+      width: 3rem;
+      height: 3rem;
+      margin-bottom: 1.5rem;
+    }
+    #warningHeading {
+      text-align: center;
+    }
+    #warningContainer p {
+      font-size: 1.2rem;
+    }
+    #warningContainer a {
+      margin-bottom: 1rem;
+    }
+  </style>
+  <div id="warningContainer" class="container">
+    ${warningIcon}
+    <h1 id="warningHeading">Warning</h1>
+    <p>
+      If you have your Internet Identity tied to only one device and you lose
+      that one device, you will be locked out.
+    </p>
+    <p>
+      Additionally on iOS and MacOS clearing your Safari browser data will
+      <em>delete</em> your logins from the device.
+    </p>
+    <p>
+      Hence as a best practice we always recommend you add multiple devices to
+      your Internet Identity and at <em>least</em> one actual security key. You
+      can see how to add more devices here:
+    </p>
+    <a
+      target="_blank"
+      href="https://sdk.dfinity.org/docs/ic-identity-guide/auth-how-to.html"
+      >https://sdk.dfinity.org/docs/ic-identity-guide/auth-how-to.html</a
+    >
+    <p>
+      For now, the best recovery strategy is to add extra devices or a Yubikey
```

Comment:
```suggestion
      For now, the best recovery strategy is to add extra devices or a YubiKey
```
Maybe we shouldn't single out one product?

Reply:
+1, we don't want to use a brand name in our UI. There could be legal repercussions.

Reply:
Also, I'd replace "or" with "and"

Reply:
Full suggestion: 
"Currently the best strategy to ensure you can log in is to add extra devices and at least one security key..."